### ISSUE TITLE
ERROR: Error opening URL 

### ISSUE THREAD

**Issue by j-ktz** [CodyBerenson/PGMA-Modernized#210]
Hey Boys

Been getting this error lately. Looks like  maybe an issue between Plex and the agents?

[com.plexapp.agents.QueerClick.log](https://github.com/CodyBerenson/PGMA-Modernized/files/10328032/com.plexapp.agents.QueerClick.log)
[com.plexapp.agents.WayBig.log](https://github.com/CodyBerenson/PGMA-Modernized/files/10328033/com.plexapp.agents.WayBig.log)
[com.plexapp.agents.Fagalicious.log](https://github.com/CodyBerenson/PGMA-Modernized/files/10328034/com.plexapp.agents.Fagalicious.log)

**Comment by JPH71** [comment]
Hi J-ktz

Could you scrape the film scene against the agent, you based it's name
on....
So I can see exactly why's its failing....

**Comment by j-ktz** [comment]
Sure, I just tried on these two

Waybig: (FraternityX) - PRETTY BOY POUNDED (Seth Knight, Bentley, Carter Michaels, Dean, Rocky And Travis) (Bareback) (2017)
F*glicious: (Next Door Taboo) - Dallas Preston takes Jayden Marcos’ big cock raw in “Stepbrothers With Benefits” (2022)

[com.plexapp.agents.WayBig.log](https://github.com/CodyBerenson/PGMA-Modernized/files/10328104/com.plexapp.agents.WayBig.log)
[com.plexapp.agents.Fagalicious.log](https://github.com/CodyBerenson/PGMA-Modernized/files/10328105/com.plexapp.agents.Fagalicious.log)

**Comment by JPH71** [comment]
I will test them tomorrow...
Getting drunk as we are now I to our new year...

**Comment by j-ktz** [comment]
Cheers! Have one for me! 🥂 Sent from my iPhoneOn Dec 31, 2022, at 6:27 PM, JPH71 ***@***.***> wrote:﻿
I will test them tomorrow...
Getting drunk as we are now I to our new year...

**Comment by JPH71** [comment]
I am on number six and cross eyed!!!!
Passion fruit flavoured gin is bliss!!!!!!!!!!!!
Happy New Year to you when it comes to you!!!

**Comment by j-ktz** [comment]
Hope you survived the new year! Give me a shout if you need me to pull any more logs or anything. I think this one migth be tricky.

**Comment by JPH71** [comment]
Looking throught the logs:
**WAYBIG**
(Sean Cody) - Eddie Burke Fucks Phoenix (2022) is what it should be named to be found on WayBig - You have it as **(Sean Cody) - Eddie Burke fucks Phoenix raw (2022).mp4**
(TimTales) - Tommy Dreams And Diego Mattos DP Valentin Amour In ‘L’amour À Trois’ (2022) - You have it as **(TimTales) - Valentin Amour gets double-penetrated by Tommy Dreams and Diego Mattos’ 9-inch cocks in “L’amour à trois” (2022)** 

**QUEERCLICK**
(Next Door Taboo) - Dallas Preston takes Jayden Marcos’ big cock raw in “Stepbrothers With Benefits” (2022) should be **(Next Door Taboo) - Jayden Marcos Fucks Dallas Preston – Stepbrothers With Benefits (2022)**

**FAGALICIOUS**
(Next Door Taboo) - Dallas Preston takes Jayden Marcos’ big cock raw in “Stepbrothers With Benefits” (2022)
I looked at your log file and it seems to replace all the quotes and double quotes with an "a"....
So I tested this on my system to see if I could replicate this behaviour....
Fortunately for me it scraped with no issues...
Attached is the log file....

[com.plexapp.agents.Fagalicious.log](https://github.com/CodyBerenson/PGMA-Modernized/files/10369480/com.plexapp.agents.Fagalicious.log)

**I suggest that you:
1.  delete all your agent bundles and replace them with the latest from the repository, then set their preferences including the plex tokens if it does not retrieve it automatically.
2. Check the initial logs that will appear to see if they pass the scrape activation tests then retest....
3.  test Fagalicious with _(Next Door Taboo) - Dallas Preston takes Jayden Marcos’ big cock raw in “Stepbrothers With Benefits” (2022)_
4. Check in the Logs for the value of IAFDSearchTitle and see if it replaces the quote marks with the letter "a"
**

Jason xxxx

**Comment by j-ktz** [comment]
Morning sir! It looks like Waybig did match the Sean Cody scene correctly. Yay! I tested the Next Door scene with Fagalicious and it does replace quotes with "a". Just like you thought. See log attached. 
[com.plexapp.agents.Fagalicious.log](https://github.com/CodyBerenson/PGMA-Modernized/files/10373556/com.plexapp.agents.Fagalicious.log)

**Comment by j-ktz** [comment]
Hey @JPH71,

I've continued to do some testing (avoid Fagalicious because of all the quotes) and using WayBig and QueerClick where I can. Here are some issues I'm still having with matching:

WayBig:
(ChaosMen) - Aries, Solomon & Vander (Tag-Team Raw) (Director’s Cut) (2013)
(ChaosMen) - Phineas & Vander ‘Edge’ (RAW PISS) [Director’s Cut] (2016)
(Bel Ami Online) - Joel Birkin & Jack Harrer Fuck Arne Coen In ‘Offensively Large, Part 4’ (2016)
(Bel Ami Online) - Orri Aasen Bottoms For Ryan McKenna In ‘Summer Loves’ (2020)
(Next Door Studios) - Jayden Marcos, Kane Fox And Nico Coopa In ‘Fun On The Job’ (2022)
(Fuckermate) - Vadim Romanov Fucks Ricky Hard In ‘Bare Break’ (2022)

QueerClick:
(Falcon Studios) - Bruce Beckham, Taylor Reign, and Drake Von - Tales From The Locker Room 3 (2023)
(Rod’s Room) - Trevor Brooks and Evan Knoxx (2023)

[com.plexapp.agents.QueerClick.log](https://github.com/CodyBerenson/PGMA-Modernized/files/10397133/com.plexapp.agents.QueerClick.log)
[com.plexapp.agents.WayBig.log](https://github.com/CodyBerenson/PGMA-Modernized/files/10397134/com.plexapp.agents.WayBig.log)

**Comment by JPH71** [comment]
Let me know the issues with fagzlicious also.. in relation to the quotes...

**Comment by JPH71** [comment]
I can't seem to download the log files you sent.....
send them again...
also send one from fagalicious that's misbehaving, then i can sort them all
out!

**Comment by JPH71** [comment]
It finally downloaded...
But you are scraping a number of film scenes at the same time and
unfortunately as the scraping process runs in parallel - the logs get
jumbled up.
open the log file... delete its contents and save. Then run a refresh
metadata on the film scene that is giving you the issue. Then send that to
me... I can then track what is going on and at what stage you get your
issue....

Jason xxx

**Comment by j-ktz** [comment]
Sure thing. I scrapped a couple of Fagalicious ones this am. These all failed:

(Lucas Entertainment) - Kosta Viking pounds Alex Cabrera in “Bareback Auditions 21 - Thoroughly Bred” (2022)
(NakedSword Originals) - Beau Butler takes nine big dicks raw in Blame It On Rio - Beau's First Gangbang (2022)

The Naked Sword one turned the quotes into these weird horizontal lines when I scraped them. LOL?

I noticed it here too:

(CutlersDen) - Cutler X pounds Bobby Day bareback in Boss Like Me (2023)

Also, how do we handle "/" in the username for example [Dom/Sub]. I tried replacing it with hyphen but that didn't seem to work. 

I went back to the WayBig ones and some of them scraped successfully after I swapped  out curly quotes for straight quotes (wow homophobic lol)

[com.plexapp.agents.Fagalicious.log](https://github.com/CodyBerenson/PGMA-Modernized/files/10402164/com.plexapp.agents.Fagalicious.log)

**Comment by JPH71** [comment]
I will sort out the / as its an illegal filename character..... and that is
the first one I have seen!
I hate unicode characters...... it was turning quotes into an "a" for
another user!!!!

Jason

**Comment by j-ktz** [comment]
Thank you sir!

**Comment by JPH71** [comment]
Could you send me the link with the DOM?SUB in the title - I will work on this....
I will make it convert the / into a hyphen (minus) and get it too match

**Comment by j-ktz** [comment]
Yes sir! Here you are: https://www.waybig.com/blog/2023/01/06/cockyboys-theo-brady-fucks-trevor-brooks-dom-sub-action/

**Comment by JPH71** [comment]
Thanks Sweet Pea!

**Comment by JPH71** [comment]
Tried these 2
Waybig: (FraternityX) - PRETTY BOY POUNDED (Seth Knight, Bentley, Carter
Michaels, Dean, Rocky And Travis) (Bareback) (2017)
F*glicious: (Next Door Taboo) - Dallas Preston takes Jayden Marcos’ big
cock raw in “Stepbrothers With Benefits” (2022)

Both worked!

**Comment by JPH71** [comment]
I have tried these and they have scraped!!!

WayBig:
(ChaosMen) - Aries, Solomon & Vander (Tag-Team Raw) (Director’s Cut) (2013)
(Next Door Studios) - Jayden Marcos, Kane Fox And Nico Coopa In ‘Fun On The
Job’ (2022)

QueerClick:
(Falcon Studios) - Bruce Beckham, Taylor Reign, and Drake Von - Tales From
The Locker Room 3 (2023)

Are you still having issues with these??

**Comment by JPH71** [comment]
Could you send me the link to the title with the "DOM/SUB"
I can't seem to find the title .......

Cheers

Jason

**Comment by JPH71** [comment]
Ignore it - I found it... its a waybig title!

**Comment by JPH71** [comment]
Could you try typing in the double quotes for the titles which were failing and let me know if they are still been converted to an **a**

**Comment by JPH71** [comment]
Code for Dom/Sub with Cody at the moment

**Comment by JPH71** [comment]
After testing - please mark this as closed if all works for you...
For the issues that still fail - please re-open another issue - so that I can track it more effciently...

Jason

**Comment by JPH71** [comment]
Darned it...
I have to pur in acomditional import...
As soon as I have done this I will send the code to Cody...
Thanks for the swift response...

**Comment by JPH71** [comment]
Hi Jesse

Code is with Cody now!

Cheers

Jason

**Comment by JPH71** [comment]
Please mark as closed!

**Comment by JPH71** [comment]
Could you mark this as closed!

**Comment by CodyBerenson** [comment]
Ok, I just closed it.

p.s.  charge the newbie a consulting fee  ;)  either in cash, or Skype
service, or selfies....or he can amazon liquor to you!  we can't do that
here, but i do it in Spain all the time....so i'm assuming it can be done
in Belgium?
